projects/ion/src/lib/indicator/indicator.component.html

```diff
@@ -39,9 +39,17 @@ <h1 data-testid="ion-indicator-title">{{ title }}</h1>
           color="primary"
         ></ion-spinner>
       </div>
-      <header *ngIf="!loading && !error">
-        <h4 data-testid="ion-indicator-value">{{ value }}</h4>
-        <p data-testid="ion-indicator-second-value">{{ secondValue }}</p>
+      <header *ngIf="!loading && !error" [class.no-data]="isNoData()">
+        <ng-container *ngIf="!isNoData()">
```

Comment:
```suggestion
        <ng-container *ngIf="value === null || value === undefined || value === ''">
```
Calling a function on HTML you loss performance, my suggestion is or call your args on ngIf

https://medium.com/javarevisited/why-you-should-not-use-functions-in-angular-html-f445371a4b6b